src/core_seaice/shared/mpas_seaice_velocity_solver.F

```diff
@@ -2816,12 +2823,12 @@ subroutine solve_velocity_revised(domain)
 
              ! right hand side of matrix solve
              rightHandSide(1) = stressDivergenceU(iVertex) + airStressVertexU(iVertex) + surfaceTiltForceU(iVertex) + &
-                  oceanStressCoeff(iVertex) * oceanStressU(iVertex) + &
+                  oceanStressCoeff(iVertex) * oceanStressU(iVertex) + bergStressU(iVertex) + &
```

Comment:
The terms bergStressU, bergStressV are in units kg*m*s^{-2}, and I believe this (rightHandSide) is the appropriate place to put them.

These terms are calculated when namelist config_berg_seaice_interaction = .true., and are calculated in the subroutine seaice_run_berg_postdynamics.